### DISCUSSION TITLE
Feature request: Allow workflow to receive an upload of file as user input in GitHub Actions

### DISCUSSION THREAD

**Discussion by YgorAraujo** [community/community#37527] [Actions]
I would like to upload a file to a workflow in GitHub Actions. 

I have done a search about this question, but I found that input parameters in a workflow have only a few types, like text, number, boolean, dropdown, etc. To summarize, I didn't find a type of input to upload a file as input.

If the workflow in GitHub Actions have a type of input that enable upload a file as input, it would save a lot of time.

**Comment by lopeta01** [comment]
Agreed, this feature would be very useful.

**Comment by ossanna16** [comment]
Hi there @YgorAraujo and welcome to our community! Thank you for the feedback 🙂
